core/src/main/java/ai/timefold/solver/core/config/util/ConfigUtils.java

```diff
@@ -384,6 +392,7 @@ public static List<Member> getAllMembers(Class<?> baseClass, Class<? extends Ann
         return memberStream.distinct().sorted(alphabeticMemberComparator).collect(Collectors.toList());
     }
 
+    // TODO
     @SafeVarargs
     public static Class<? extends Annotation> extractAnnotationClass(Member member,
             Class<? extends Annotation>... annotationClasses) {
```

Comment:
@triceo From the function code, it seems to me the parameters here should all be `@NonNull`, is that right?